sources/packages/web/src/components/aest/AssessApplicationOfferingChangeRequestModal.vue

```diff
@@ -36,16 +44,33 @@ import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
 import { ref, defineComponent, computed } from "vue";
 import { useModalDialog, useRules } from "@/composables";
 import { ApplicationOfferingChangeAssessmentAPIInDTO } from "@/services/http/dto";
-import { ApplicationOfferingChangeRequestStatus, VForm } from "@/types";
+import { ApplicationOfferingChangeRequestStatus, VForm, Role } from "@/types";
+import CheckPermissionRole from "@/components/generic/CheckPermissionRole.vue";
 
 export default defineComponent({
   components: {
     ModalDialogBase,
+    CheckPermissionRole,
+  },
+  props: {
+    processing: {
+      type: Boolean,
+      required: true,
+      default: false,
+    },
   },
   setup() {
     const note = ref("");
-    const { showDialog, showModal, resolvePromise, showParameter } =
-      useModalDialog<ApplicationOfferingChangeAssessmentAPIInDTO | boolean>();
+    const {
+      showDialog,
+      showModal,
+      resolvePromise,
+      showParameter,
+      hideModal,
+      setLoading,
+      getLoading,
+      loading,
```

Comment:
for my info, what is the difference between `getLoading` and `loading`

Reply:
@sh16011993 do we need both?

Reply:
Oh, my apologies. I was doing some test runs with `getLoading`. Removed it.